doc/reference/reports.rst

```diff
@@ -35,15 +35,18 @@ can take the following attributes:
 
 ``id``
     the generated record's :term:`external id`
-``name`` (mandatory)
-    only useful as a mnemonic/description of the report when looking for one
-    in a list of some sort
+``string`` (mapping to ``IrActionsReport.name`` field)
+    If ``print_report_name`` isn't specified, will be used as file name.
```

Comment:
In `actions.rst` the default behavior and relationship with `name` / `string` is mentioned in `print_report_name`'s description while here it's in `string` instead of `print_report_name`. I think it would be clearer if the two pages were consistent in that, because I had to do a double-take to make sure the default behavior wasn't different between the two.